src/geo/transform.js

```diff
@@ -314,11 +314,17 @@ class Transform {
         return this.tileSize * this.scale;
     }
 
+    // $FlowFixMe This getter returns the incorrect value. See free_camera.getDistanceToElevation for the rationale.
     get cameraWorldSize(): number {
         const distance = Math.max(this._camera.getDistanceToElevation(this._averageElevation), Number.EPSILON);
         return this._worldSizeFromZoom(this._zoomFromMercatorZ(distance));
     }
 
+    get cameraWorldSize_(): number {
```

Comment:
This duplicate function seems potentially confusing. Would it be possible to tweak fog to effectively stay the same, while still applying the conversion correctly?

If that's not possible, thoughts on giving the old function a more specific name, e.g. `cameraWorldSizeForFog?`

Reply:
Can we please agree to not add yet another multiplication factor? We already have a few that IMHO are causing lot more confusion. Therefore I am going to rename it to `cameraWorldSizeForFog` as you suggested.

Reply:
Sorry made a blunder in the naming. How about `cameraWorldSizeForSymbols` instead for the new function just to keep the scope small and avoid updating many files.

Reply:
I meant in many places*. Indeed it's only in `transform.js`. No strong preference here so I'll update it.